src/ray/raylet/test/cgroup_v2_utils_test.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/ray/raylet/cgroup_v2_utils.h"
+
+#include <gtest/gtest.h>
+
+namespace ray {
+
+namespace {
+
+// Precondition: cgroup V2 has already been mounted as rw.
+//
+// Setup command:
+// sudo umount /sys/fs/cgroup/unified
+// sudo mount -t cgroup2 cgroup2 /sys/fs/cgroup/unified -o rw
```

Comment:
qq, I am not familiar but what requires to put these step in ray's cicd?

Reply:
Don't think it should be placed under CI, which is a group of **unit test**, but for cgroup related feature they should be tested as **integration tests**.

Reply:
That's why you see me mark test as "manual".